schema.sql

```diff
@@ -0,0 +1,32 @@
+DROP DATABASE IF EXISTS reviewsDb;
+CREATE DATABASE reviewsDb;
+USE reviewsDb;
+
+CREATE TABLE restaurant (
+  restaurant_id INT NOT NULL AUTO_INCREMENT,
+  name VARCHAR(50) NOT NULL,
+  PRIMARY KEY (restaurant_id)
+);
+
+CREATE TABLE user (
+  user_id INT NOT NULL AUTO_INCREMENT,
+  username VARCHAR(50) NOT NULL,
```

Comment:
What is the username if they provide an anonymous review - if null, why is this 'NOT NULL'?